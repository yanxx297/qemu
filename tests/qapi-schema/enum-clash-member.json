# FIXME: we should reject enums where members will clash when mapped to C enum
{ 'enum': 'MyEnum', 'data': [ 'one', 'ONE' ] }
