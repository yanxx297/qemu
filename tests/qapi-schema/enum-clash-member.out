[OrderedDict([('enum', 'MyEnum'), ('data', ['one', 'ONE'])])]
[{'enum_name': 'MyEnum', 'enum_values': ['one', 'ONE']}]
[]
